packages/v4/src/content/design-guidelines/usage-and-behavior/dashboard/dashboard.md

```diff
@@ -0,0 +1,335 @@
+
+You can rearrange cards multiple times until you find the layout that works:
+
+<img src="./img/dashboard-iterations.png" alt="dashboard layout iterations" width="5450"/>
+
+We knew details and performance were our top priority items, so we kept those up top and mainly iterated around the rest of the cards. Note how we changed the height of some of the cards to better fit the layout.
+
+Dynamic cards deserve special considerations. You want to avoid situations such as having a one column dynamic card expand and push down a two column card below it, creating a gap of unused space. We’ll explore how this might play out with ther events card in our next example.
+
+<img src="./img/dashboard-what-to-avoid.png" alt="dashboard layout with a gap" width="1680"/>
+
+We could solve this gap by setting the inventory card height to always match the events card. Another potential solution is to always have your events card at a defined height, introducing scroll for any information beyond that does not fit. Flexible and accommodating layouts usually provide the most effective long term solution.
+
+### Step 5: Add your content and refine your layout
+
+Once you’ve found a couple layouts you’re comfortable with, add your actual content designs to each card and see how it affects their sizing. Start with the standard card templates [link] described at the start of this document. You can modify these or create custom card layouts to meet your needs. Let’s see how we can populate each of the cards in our preliminary layout.
+
+#### Details card
+Recall that in Step 1, we defined the Details card as needing to display the cluster name, the provider, and the OS version. This can easily be accomplished by using a standard details card.
+
+<img src="./img/details-card-example.png" alt="detail card content" width="439"/>
+
+#### Performance card
+The performance card needs to display current usage and usage history over time for five different measures: IOPS, Latency, Throughput, Recovery rate, and Recovery IOPS. For this use case, we can group five standard trend cards and arrange them in a horizontal grid. Then, we can add a filter in the header to change the scope of the data that we wish to measure.
+
+<img src="./img/performance-card-example.png" alt="performance card content" width="1185"/>
+
+#### Health card
+The health card will be a custom card to show the state of the cluster and the cluster data. While we are not using a standard card here, we can still borrow elements of the aggregate status card for displaying overall health.
+
+<img src="./img/health-card-example.png" alt="health card content" width="670"/>
```

Comment:
The "see all" link is still in overpass font. Also looking at this again, would change the info icon to our unfilled question-circle icon which we now recommend using in these situations (over an info icon which we want to reserve for status situations)